src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Automation/Peers/ItemAutomationPeer.cs

```diff
@@ -338,13 +338,30 @@ internal static int GetPositionInSetFromItemsControl(ItemsControl itemsControl,
         {
             int position = AutomationProperties.AutomationPositionInSetDefault;
             ItemCollection itemCollection = itemsControl.Items;
-            position = itemCollection.IndexOf(item);
 
             if (itemsControl.IsGrouping)
             {
                 int sizeOfGroup;
                 position = FindPositionInGroup(itemCollection.Groups, position, out sizeOfGroup);
```

Comment:
This line will be affected by the deleted line above unless I'm missing something. If I read your comment, I understand it was not your intention though

Reply:
Good catch - did not realize this variable was used on this line. Re-added the deleted line.